experimental/ssh/internal/server/authorized_keys.go

```diff
@@ -0,0 +1,63 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/databricks/cli/experimental/ssh/internal/keys"
+	"github.com/databricks/cli/libs/log"
+	"github.com/databricks/databricks-sdk-go"
+)
+
+type AuthorizedKeysManager struct {
```

Comment:
Consider adding a comment documenting the lifecycle of the keys and when they are added